WISE-Community/WISE-API
# feat(Peer Group Activity): Move settings to project level

## Changes
- Introduce new "peerGroupActivity" an array to store PeerGroupActivities in the unit, at the project level.

## Test
- Open AT for a run unit, and open the advanced -> JSON editing view for the entire unit
- Add peerGroupActivities field at the top-level (same level as "title" and "nodes"):
```
   "title": "GCC",
...
   "peerGroupActivities": [
      {"tag":"peerGroupActivity1","logic":"manual"},
      {"tag":"peerGroupActivity2","logic":"manual"},
      {"tag":"peerGroupActivity3","logic":"manual"}],
...
```
- When the JSON saves, it should add rows in peer_group_activities table "peerGroupActivity1", "peerGroupActivity2", "peerGroupActivity3", each with "manual" logic.
- Creating a new run with this new "peerGroupActivities" field should create new entires in the peer_group_activities table as well
- Editing unit (with and without run) that does not have this new "peerGroupActivities" field should work as before.

Closes #106
